src/styles/blocks/stars--2.css

```diff
@@ -0,0 +1,44 @@
+.stars--2 {
+  display: flex;
+  margin-top: 4px;
+}
+
+.stars--2 .stars__star:nth-child(1) {
+  display: block;
+  background-image: url(../../images/star-active.svg);
+  width: 13px;
+  height: 12.2px;
+  margin-left: 22px;
```

Comment:
The first star has a different margin-left than the others. If this is intentional for design purposes, it's fine; otherwise, consider keeping a consistent spacing between stars for a uniform look.